perception/traffic_light_classifier/README.md

```diff
@@ -1,3 +1,5 @@
+<!-- cspell: ignore mmclassification mmcv -->
```

Comment:
I think you can add `MMCV` to the dictionary.
```suggestion
<!-- cspell: ignore mmclassification -->
```
https://github.com/open-mmlab/mmcv

Reply:
https://github.com/tier4/autoware-spell-check-dict/pull/505

Reply:
I removed mmcv from ignore.
And thanks a lot for PR below. 
https://github.com/tier4/autoware-spell-check-dict/commit/c78b32e5dfa46f37fd7d414321ee93258c25b354

Reply:
And `mmclassification` seems to be also okay.
https://mmclassification.readthedocs.io/en/latest/getting_started.html

Reply:
Thank you for https://github.com/tier4/autoware-spell-check-dict/commit/de0ea2b7f44992e986137ccac54eb7f428e882d2